tests/python/pytorch/graphbolt/impl/test_legacy_dataset.py

```diff
@@ -0,0 +1,34 @@
+import dgl.graphbolt as gb
+import pytest
+import torch
+from dgl import AddSelfLoop
+from dgl.data import AsNodePredDataset, CoraGraphDataset
+
+
+def test_LegacyDataset_homo_node_pred():
+    cora = CoraGraphDataset(transform=AddSelfLoop())
```

Comment:
For ogb datasets, is it possible to transform the graph when generating gb.Dataset?